lact-daemon/src/server/gpu_controller.rs

```diff
@@ -131,16 +133,16 @@ pub struct PciSlotInfo {
 }
 
 #[cfg(feature = "nvidia")]
-pub type NvidiaLibs = (Rc<Nvml>, Rc<Option<NvApi>>);
+pub type NvidiaLibs = (Arc<Nvml>, Arc<Option<NvApi>>);
 #[cfg(not(feature = "nvidia"))]
 pub type NvidiaLibs = ();
 
 pub(crate) fn init_controller(
     path: PathBuf,
     pci_db: &pciid_parser::Database,
-    nvml: &LazyCell<Option<NvidiaLibs>>,
-    amd_drm: &LazyCell<Option<LibDrmAmdgpu>>,
-    intel_drm: &LazyCell<Option<Rc<IntelDrm>>>,
+    nvml: &'static LazyLock<Option<NvidiaLibs>>,
+    amd_drm: &'static LazyLock<Option<LibDrmAmdgpu>>,
+    intel_drm: &'static LazyLock<Option<IntelDrm>>,
```

Comment:
As I mentioned before already, you can directly reference the statics inside of this function (after making them crate-public), there is not much point to passing them as arguments now, it would make the function cleaner